docs/quick_start.md

```diff
@@ -125,8 +125,8 @@ of seed nodes returned from `_sample_seed` function.
 
 ### Graph flow
 
-The `EgoGraph` are returned in NumPy format, 
-to use it in a deep learning framework backend such as TensorFlow, 
+The `EgoGraph` are returned in NumPy format. 
```

Comment:
Thanks for your contribution. It would be much appreciated if you could fix more typos.